Marlin/src/HAL/HAL_DUE/HAL_timers_Due.cpp

```diff
@@ -46,7 +46,6 @@
 
 #define NUM_HARDWARE_TIMERS 9
 
-#define PRESCALER 2
```

Comment:
`HAL/shared/servo_private.h` needs `PRESCALER` to be defined. It's used to define `usToTicks` and `ticksToUs`…
```cpp
#define usToTicks(_us)    (clockCyclesPerMicrosecond() * (_us) / (PRESCALER))
#define ticksToUs(_ticks) (unsigned(_ticks) * (PRESCALER) / clockCyclesPerMicrosecond())
```

Reply:
Trying the servo-oriented one first….

Reply:
You'r right...I missed it